lib/logger.js

```diff
@@ -71,6 +71,12 @@ function Message(args) {
                 result.message = result.message ? result.message += ` ${arg.message}` : arg.message;
                 delete arg.message;
             }
+            if (arg.loggingContext){
```

Comment:
I think this line needs to be changed to `if (arg && arg.loggingContext)`.  I made a recent comment to @PedroKinetica to modify the message property you borrowed above.